src/models/api_key/request.rs

```diff
@@ -1,8 +1,8 @@
+use crate::models::PermissionGrant;
 use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize)]
 pub struct ApiKeyRequest {
```

Comment:
Also maybe we should inject defaults permissions, like api key should be always able to delete itself.